lib/index.ts

```diff
@@ -226,14 +234,28 @@ export function buildArgs(
   executionPath: string,
   targetFile?: string,
   mavenArgs?: string[] | undefined,
+  mavenAggregateProject = false,
 ) {
+  let args: string[] = [];
+
+  if (mavenAggregateProject) {
+    // to workaround an issue in maven-dependency-tree plugin
+    // when unpublished artifacts do not exist in either a local or remote repository
+    // see https://stackoverflow.com/questions/1677473/maven-doesnt-recognize-sibling-modules-when-running-mvn-dependencytree
+    args = args.concat('test-compile');
+  }
```

Comment:
can we have an else block here instead of the if statement on line 255?

Reply:
We need to concat this option after mvn